edge-info/edge-info

```diff
@@ -368,7 +368,6 @@ firmware(){
     local bito
     bito=$(getconf LONG_BIT)
     kernelV=$(uname -r)
-    echo "${NORM}"
```

Comment:
or change it to echo -e to  "enable interpretation of backslash escapes"

Reply:
The best code line is the removed one.